server/views/formPages/incident/report-cancelled.html

```diff
@@ -0,0 +1,18 @@
+{% extends "../../partials/layout.html" %} 
+{% from "govuk/components/button/macro.njk" import govukButton %} 
+{% set pageTitle = pageData.pageTitle %} 
+
+{% block content %}
+  <div class="govuk-body">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl mainHeading">{{pageData.pageTitle}}</h1>
+        <p>You may be asked to complete a statement if you are named as being involved in the incident.</p>
+          <p>
+            <a href="{{ links.exitUrl }}" class="govuk-link govuk-!-font-size-19" data-qa="exit-to-dps-link">Return to Digital Prison Services</a>
+          </p>
+
+      </div>
+    </div>
+  </div>
+{% endblock %} {% block script %} {% endblock %}
```

Comment:
formatting is screwed up again. Do you need the script block or can that be removed?

Reply:
done